topo_processor/stac/asset.py

```diff
@@ -41,7 +41,7 @@ def get_content_type(self) -> str:
             return self.content_type
         return MimeTypes().guess_type(self.target if self.target else self.source_path)[0]
 
-    def get_checksum(self) -> str:
+    def get_checksum(self) -> Any:
```

Comment:
So, there seems to be three options we could pick from and mypy passes?
Is there a rule or best practise for how we should choose?
1. `-> Any`
2. `-> Union[str, Any]`
3. `-> str...
     ... return cast(str, value)`

Reply:
Good spot. It should be `Union[str, Any]` as that is what returns `multihash_as_hex`